admin/app/components/team/list.js

```diff
@@ -34,6 +34,12 @@ export default class List extends Component {
   @action
   async updateMemberRole(adminMember) {
     const previousRole = adminMember.role;
+
+    if (!this.newRole) {
```

Comment:
**remarque**
Si je me co avec superadmin et que je lui met le rôle de certif (sans valider) et que je remet le rôle de superadmin : la requête API se fait et la notif verte pop ... 🤔

Reply:
du coup le front récupère le newRole parce que tu l'as selectionné donc oui. 
Là le cas que je protège c'est quand tu veux modifier et que tu valides sans toucher à la liste des rôles

Reply:
on pourrait avoir un genre

```suggestion
    if (!this.newRole || this.newRole === previousRole) {
```

Reply:
encore mieux ! c'est ajouté